docs/developers/testing.md

```diff
@@ -53,6 +53,45 @@ If you'd like to include them in local tests, set the environment variable "CI":
 CI=1 pytest
 ```
 
+It is also possible to run test using `tox`. This is the same way as it is done in CI.
+The main difference is that tox will create a virtual environment for each test environment, so it will take more time
+but it will be more similar to the CI environment.
+
+```sh
+tox -e py310-linux-pyqt5
+```
+
+To get list of all available environments run:
+
+```sh
+tox list
+```
+
+### Running tests without jumping windows
+
+Part of tests are opening and closing windows. This can be annoying if you are doing something when wait on the test end. 
```

Comment:
```suggestion
Some tests create visible napari viewers, which pop up on your monitor then quickly disappear.
This can be annoying if you are trying to use your computer while the tests are running.
```